tests/foreman/longrun/test_oscap.py

```diff
@@ -94,86 +70,118 @@ def fetch_scap_and_profile_id(cls, scap_name, scap_profile):
+        request.cls.puppet_env = request.cls.puppet_env.update(['organization'])
         smart_proxy = (
             entities.SmartProxy().search(query={'search': f'name={sat6_hostname}'})[0].read()
         )
-        smart_proxy.import_puppetclasses(environment=cls.puppet_env.name)
-        env = entities.LifecycleEnvironment(organization=org, name=gen_string('alpha')).create()
+        smart_proxy.import_puppetclasses(environment=request.cls.puppet_env.name)
+        request.cls.env = entities.LifecycleEnvironment(
+            organization=module_org, name=gen_string('alpha')
+        ).create()
         # Create content view
-        content_view = entities.ContentView(organization=org, name=gen_string('alpha')).create()
+        request.cls.content_view = entities.ContentView(
+            organization=module_org, name=gen_string('alpha')
+        ).create()
         # Create activation keys for rhel6, rhel7 and rhel8.
         for repo in repo_values:
             activation_key = entities.ActivationKey(
-                name=repo.get('akname'), environment=env, organization=org
+                name=repo.get('akname'), environment=request.cls.env, organization=module_org
             ).create()
             # Setup org for a custom repo for RHEL6, RHEL7 and RHEL8.
             setup_org_for_a_custom_repo(
                 {
                     'url': repo.get('repo'),
-                    'organization-id': org.id,
-                    'content-view-id': content_view.id,
-                    'lifecycle-environment-id': env.id,
+                    'organization-id': module_org.id,
+                    'content-view-id': request.cls.content_view.id,
+                    'lifecycle-environment-id': request.cls.env.id,
                     'activationkey-id': activation_key.id,
                 }
             )
 
-        for content in cls.rhel8_content, cls.rhel7_content, cls.rhel6_content:
+        for content in rhel8_content, rhel7_content, rhel6_content:
             content = Scapcontent.info({'title': content}, output_format='json')
             organization_ids = [
                 content_org['id'] for content_org in content.get('organizations', [])
             ]
-            organization_ids.append(org.id)
+            organization_ids.append(module_org.id)
             Scapcontent.update({'title': content['title'], 'organization-ids': organization_ids})
 
-        return {
-            'org_name': org.name,
-            'cv_name': content_view.name,
-            'sat6_hostname': settings.server.hostname,
-            'ak_name': {'rhel8': ak_name_8, 'rhel7': ak_name_7, 'rhel6': ak_name_6},
-            'env_name': env.name,
-        }
-
     @pytest.mark.tier4
     @pytest.mark.libvirt_content_host
     @pytest.mark.upgrade
-    def test_positive_upload_to_satellite(self):
+    @pytest.mark.parametrize(
+        "rhel_repo, content, profile1, profile2, profile3, distro",
```

Comment:
Please switch these to single-quote strings.

Reply:
updated the pr.